plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/RetryingConnectionFactoryModule.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+import io.trino.plugin.jdbc.RetryingConnectionFactory.DefaultRetryStrategy;
+import io.trino.plugin.jdbc.RetryingConnectionFactory.RetryStrategy;
+
+import static com.google.inject.multibindings.OptionalBinder.newOptionalBinder;
+
+public class RetryingConnectionFactoryModule
```

Comment:
Do we need dedicated Module for two binds?

Reply:
It's used in two places in production and also in one test. The test actually verifies that the module is written in such a way that the override of RetryStrategy works. If it was copied-pasted inline in the production code, it would be impossible to have such test. I'm happy to do the inline copy-paste approach if you think that the 'override test' is not necessary.

Reply:
👍